utils/parachain-node/runtime/src/lib.rs

```diff
@@ -676,6 +676,25 @@ impl DenomToAssetId<Runtime> for IbcDenomToAssetIdConversion {
 	}
 }
 
+#[derive(
+	Debug, codec::Encode, Clone, codec::Decode, PartialEq, Eq, scale_info::TypeInfo, Default,
+)]
+pub struct MemoMessage;
```

Comment:
Why we don't use string in the runtime?

Reply:
Encode, Decode and TypeInfo are not implemented for Strings